### ISSUE TITLE
Google docs table navigation: text is written to the wrong cell

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#9570]
### Steps to reproduce:
1. Open a new document in Google docs
2. Enable screen reader and also braille support
3. Create a table with some rows and columns
4. Place the carret in the first cell
5. Press ctrl+alt+right arrow key
6. Type "hello"
7. Press ctrl+alt+left arrow key
8. Type "hello"

### Actual behavior:
The text is automatically moved to the first cell of the last row. In my case, the text "hellohello" is written to row 3 column 3 although I have entered it in two different cells (row 1 column 1 and row 1 column 2).

### Expected behavior:
The text should be entered in the cells reoprted by NVDA and should not be moved to the first cell of the last row.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-17240,c20a5032

#### Windows version:
Windows 10 1809 Update

#### Name and version of other software in use when reproducing the issue:
Firefox 66.0.5, chrome 74.

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes, in 2018.4.1 the same and 2019.1.1 as well.

**Comment by Adriani90** [comment]
cc: @feerrenrut, @Qchristensen what happens to the carret visually when starting typing? (steps 6 and 8)? Is it directly moving to the first cell of the last row?

**Comment by Qchristensen** [comment]
I must admit, I find navigating the table a bit flakey.  Sometimes control+alt+arrows move, sometimes they don't, and sometimes they say they do but the focus stays where it is.  I actually can't figure out why it changes.

I did almost reproduce this just now - I pressed control+alt+right arrow, and NVDA reported "column 2" but the focus stayed in column 1, the text was typed in column 1.  Then when I moved again, NVDA reported where it thought the focus had moved to but again it never left column 1.

But before that I did manage to navigate around the table, so it doesn't seem consistent.  Now it won't move out of the first cell for me though.

**Comment by Adriani90** [comment]
This is working fine now with NVDA 2020.1, Chrome 81 and Firefox 76. Closing as works for me.
